src/js_realm.hpp

```diff
@@ -253,6 +253,31 @@ inline typename T::Function Realm<T>::create_constructor(ContextType ctx) {
 
     return realm_constructor;
 }
+    
+void convert_outdated_datetime_columns(SharedRealm realm) {
```

Comment:
This needs to be `inline` or preferably `static inline` so it doesn't violate the One Definition Rule (ODR). Also, probably could take `const SharedRealm&` as its argument.